R/license_note.R

```diff
@@ -0,0 +1,96 @@
+#' Generate LICENSE.note file.
+#'
+#' LICENSE.note generated by this function contains information about Rust crate dependencies.
+#' To use this function, the [cargo-lincense](https://crates.io/crates/cargo-license) command must be installed.
+#' @param force Logical indicating whether to re-generate LICENSE.note if LICENSE.note already exists.
+#' @inheritParams register_extendr
+#' @return No return value, called for side effects.
+#' @export
+write_license_note <- function(path = ".", quiet = FALSE, force = TRUE) {
+  if (!cargo_command_available(c("license", "--help"))) {
+    cli::cli_abort(
+      c(
+        "The {.code cargo license} command is required to run the {.fun write_license_note} function.",
+        "Please install cargo-license ({.url https://crates.io/crates/cargo-license}) first."
+      ),
+      class = "rextendr_error"
+    )
+  }
+
+  manifest_file <- rprojroot::find_package_root_file("src", "rust", "Cargo.toml", path = path)
+  outfile <- rprojroot::find_package_root_file("LICENSE.note", path = path)
+
+  if (!isTRUE(force) && file.exists(outfile)) {
+    cli::cli_abort(
+      c(
+        "LICENSE.note already exists.",
+        "If you want to re-generate LICENSE.note, set `force = TRUE` to {.fun write_license_note}."
```

Comment:
regenerate is a word that does not need a hyphen between its prefix and root.

Reply:
I have no opinion on modifying this, but note that the same wording is in the existing files.

https://github.com/extendr/rextendr/blob/4d079f7638ecd723fa92102f10aba57635df7a5b/.github/workflows/test_pkg_gen.yaml#L121
https://github.com/extendr/rextendr/blob/4d079f7638ecd723fa92102f10aba57635df7a5b/R/register_extendr.R#L18
https://github.com/extendr/rextendr/blob/4d079f7638ecd723fa92102f10aba57635df7a5b/R/rextendr_document.R#L8

Reply:
This just means we were wrong before :-D Feel free to modify these occurrences or just leave them out to be fixed in some other PR.

Reply:
lol ope!

Reply:
I have replaced all.